src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -13,10 +13,8 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
         public override string? CodeBase { get { throw null; } }
         public override System.Reflection.MethodInfo? EntryPoint { get { throw null; } }
         public override string? FullName { get { throw null; } }
-        [System.ObsoleteAttribute("The Global Assembly Cache is not supported.", DiagnosticId = "SYSLIB0005", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Correct. I verified with @safern that it isn't critical that the `ref` and `src` assemblies match on overrides. But since the GenerateReferenceAssemblySource tool was generating different code, I decided to check these in so the next person didn't have to fight with them.